packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -119,6 +154,38 @@ export class ProblemContribution extends AbstractViewContribution<ProblemWidget>
         }
     }
 
+    protected handleCopy(content: string): void {
+        const cp = document.createElement('textarea');
+        cp.value = content;
+        document.body.appendChild(cp);
+        cp.select();
+        document.execCommand('copy');
+        document.body.removeChild(cp);
+    }
+
+    protected problemCopy(selection: ProblemSelection): void {
```

Comment:
Since this method is `protected` and already within the `problem-contribution`, 
the name `copy(selection: ProblemSelection)` would be sufficient.